axis_backend/apps/documents/serializers/document_serializer.py

```
@@ -0,0 +1,299 @@
+            'version',
+            'is_latest',
+            'uploaded_by_name',
+            'client_name',
+            'contract_start_date',
+            'expiry_date',
+            'is_confidential',
+            'is_expired',
+            'is_active',
+            'created_at',
+            'updated_at'
+        ]
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)